src/MSBuild.Abstractions/ProjectItemHelpers.cs

```diff
@@ -112,7 +112,7 @@ public static class ProjectItemHelpers
         /// <summary>
         /// Checks if an item is an explicit System.ValueTuple and if the given TFM correspondes with an in-box System.ValueTuple type.
         /// </summary>
-        public static bool IsExplicitValueTupleReferenceThatCanBeRemoved(ProjectItemElement item, string tfm) =>
+        public static bool IsExplicitValueTupleReferenceThatCanBeRemoved(ProjectItemElement item, string? tfm) =>
```

Comment:
I would not allow this to be `null`. I will admit that in most cases comparing anything to `null` _should_ result in false. But the code here expects to be given a valid TFM.